db/db.js

```diff
@@ -302,55 +388,67 @@ async function updateSessionStillCounter(stillcounter, sessionid, locationid) {
  *   Person hasn't been moving for a long time
  *   Total time in the bathroom exceeds a certain value
  */
-async function isOverdoseSuspected(xethru, session, location) {
-  const now = new Date()
-  const start_time_sesh = new Date(session.start_time)
+async function isOverdoseSuspected(xethru, session, location, clientParam) {
+  try {
+    const now = new Date(await getCurrentTime())
+    const start_time_sesh = new Date(session.start_time)
 
-  // Current Session duration so far:
-  const sessionDuration = (now - start_time_sesh) / 1000
+    // Current Session duration so far:
+    const sessionDuration = (now - start_time_sesh) / 1000
 
-  // threshold values for the various overdose conditions
-  const rpm_threshold = location.rpm_threshold
-  const still_threshold = location.still_threshold
-  const sessionDuration_threshold = location.duration_threshold
+    // threshold values for the various overdose conditions
+    const rpm_threshold = location.rpm_threshold
+    const still_threshold = location.still_threshold
+    const sessionDuration_threshold = location.duration_threshold
 
-  // number in front represents the weighting
-  // eslint-disable-next-line eqeqeq
-  const condition1 = 1 * (xethru.rpm <= rpm_threshold && xethru.rpm != 0)
-  const condition2 = 1 * (session.still_counter > still_threshold) // seconds
-  const condition3 = 1 * (sessionDuration > sessionDuration_threshold)
+    // number in front represents the weighting
+    // eslint-disable-next-line eqeqeq
+    const condition1 = 1 * (xethru.rpm <= rpm_threshold && xethru.rpm != 0)
+    const condition2 = 1 * (session.still_counter > still_threshold) // seconds
+    const condition3 = 1 * (sessionDuration > sessionDuration_threshold)
 
-  let alertReason = 'NotSet'
+    let alertReason = ALERT_REASON.NOTSET
 
-  // eslint-disable-next-line eqeqeq
-  if (condition3 == 1) {
-    alertReason = 'Duration'
-  }
+    // eslint-disable-next-line eqeqeq
+    if (condition3 == 1) {
+      alertReason = ALERT_REASON.DURATION
+    }
 
-  // eslint-disable-next-line eqeqeq
-  if (condition2 == 1 || condition1 == 1) {
-    alertReason = 'Stillness'
-  }
+    // eslint-disable-next-line eqeqeq
+    if (condition2 == 1 || condition1 == 1) {
+      alertReason = ALERT_REASON.STILLNESS
+    }
```

Comment:
These `==`s can be changed to `===`s like in the `isOverdoseSuspectedInnosent` function, right?